stix2/environment.py

```diff
@@ -363,8 +363,8 @@ def partial_string_based(str1, str2):
         float: Number between 0.0 and 1.0 depending on match criteria.
 
     """
-    from fuzzywuzzy import fuzz
-    result = fuzz.token_sort_ratio(str1, str2, force_ascii=False)
```

Comment:
rapidfuzz never enforces ascii

Reply:
I don't think rounding the result is neccesary.

Reply:
This is done because fuzzywuzzy returns only full numbers between 0 and 100 (rounds the results before returning), which is not done by rapidfuzz (returns a float between 0 and 100).
So when this is not required, the rounding can be  removed.